src/FieldWarning/Assets/Units/VehicleBehaviour.cs

```diff
@@ -26,6 +26,8 @@ public class VehicleBehaviour : UnitBehaviour
     private float _terrainTiltForward, _terrainTiltRight;
     private float _terrainHeight;
 
+    
+    
     new void Awake()
```

Comment:
This kind of thing is a danger point of using git add -u too much. Please try to only commit meaningful changes. You can use `git diff --staged` before committing or `git show` after committing to do a once-over of what you committed.